packages/ui/src/proposals/components/ProposalList/ProposalList.tsx

```diff
@@ -24,9 +28,10 @@ export const ProposalList = ({ proposals, isPast }: ProposalListProps) => {
   return (
     <RowGapBlock gap={4}>
       <ProposalsListHeaders $colLayout={ProposalColLayout}>
-        <ProposalListHeader />
+        {getSortProps ? <SortHeader {...getSortProps('title')}>Title</SortHeader> : <ProposalListHeader />}
         <ProposalListHeader>Stage</ProposalListHeader>
         <ProposalListHeader>Proposer</ProposalListHeader>
+        {isPast && getSortProps ? <SortHeader {...getSortProps('createdAt')}>Ended</SortHeader> : null}
```

Comment:
```suggestion
        {isPast && getSortProps ? <SortHeader {...getSortProps('statusSetAtTime')}>Ended</SortHeader> : null}
```

Reply:
Thanks for the feedback, for me it was not very nice to have the dropdown as a column header, not very clean for me, also because we do not have any other places with a similar design. The dropdown we have in bounties but it is not a column header, but a separate UI element. If we can find a proper place to show the dropdown in the past proposals it would be great. In the meanwhile, i wil try the suggested approach and maybe will come up with something else :)

Reply:
@oleksanderkorn yes I agree it's not an ideal design, I just have a very slight preference for the dropdown over the duplicated end date because I find it nicer on the user to not have redundant information thrown at them, and it leaves some space for the eventual [Preview rationale in proposal lists #2771](https://github.com/Joystream/pioneer/issues/2771). That said I guess removing the "Ended at dd/mm/yy" from the title and just showing the date (without the "Ended at") in the `ENDED` column could work too